finite_difference/src/field_mod.f90

```diff
@@ -181,7 +181,7 @@ module field_mod
   function r2d_field_constructor(grid,    &
```

Comment:
Not the fault of this PR but I noticed the compiler complains that `max_tile_width` and `max_tile_height` are declared (lines 127-8) but not used. Please could you remove them altogether.

Reply:
Deleted these and a couple more unused variables.